examples/eSealing/src/eSealing.tsx

```diff
@@ -155,8 +156,8 @@ export default function SEALING(props: WalletConnectionProps) {
 
     useEffect(() => {
         // View file record.
-        if (connection && account && fileHashHex !== '') {
-            withJsonRpcClient(connection, (rpcClient) => viewFile(rpcClient, fileHashHex))
+        if (grpcClient && account && fileHashHex !== '') {
```

Comment:
I know this it isn't part of this change, but should `account` be included here (and in the dependency list)? It's not being used inside.

Reply:
I removed the `account` as a dependency. It is not needed here.